app/Composers/Modules/ComponentsComposer.php

```diff
@@ -33,11 +53,28 @@ class ComponentsComposer
     public function compose(View $view)
     {
         // Component & Component Group lists.
-        $usedComponentGroups = Component::enabled()->where('group_id', '>', 0)->groupBy('group_id')->pluck('group_id');
-        $componentGroups = ComponentGroup::whereIn('id', $usedComponentGroups)->orderBy('order')->get();
-        $ungroupedComponents = Component::enabled()->where('group_id', 0)->orderBy('order')->orderBy('created_at')->get();
+        $groupedComponents = $this->getVisibleGroupedComponents();
```

Comment:
This variable name is slightly confusing. `groupedComponents` makes it sound like it's a `Component` collection full of components that are in groups. It's actually a `ComponentGroup` collection, though, right?